otterdog/eclipse-kura.jsonnet

```
@@ -148,12 +148,20 @@ orgs.newOrg('iot.kura', 'eclipse-kura') {
       ],
     },
     orgs.newRepo('kura-apps') {
-      allow_merge_commit: true,
+      allow_merge_commit: false,
       allow_update_branch: false,
```

This is a nice quality of life improvement. Set it to `true` please
```suggestion
      allow_update_branch: true,
```